extensions/answer_summarizers/models_test.py

```diff
@@ -34,455 +46,320 @@ def _create_sample_answer(
             'classification_categorization': classify_category,
         }
 
-    def _create_sample_answers(
-            self, repeated_answer, times_spent_in_card, session_ids,
-            repeat_count):
-        """This is similar to _create_sample_answer, except it repeats a single
-        answer N times. It reuses times_spent_in_card and session_ids
-        cyclically as it constructs the list of sample answers.
+    def _create_answer_dicts_list(
+            self, answers, times_spent_in_card, session_ids,
+            classify_categories=(exp_domain.EXPLICIT_CLASSIFICATION,),
+            num=None):
+        """This is similar to _create_answer_dict, except it provides a list
```

Comment:
Please add docstring for args. Slightly confused because 'answers' seems to be a list here but is passed in as a string in other places.

Reply:
Done.